internal/logging/rollbar.go

```diff
@@ -8,17 +8,21 @@ import (
 
 	"github.com/rollbar/rollbar-go"
 
-	"github.com/ActiveState/cli/internal/config"
 	"github.com/ActiveState/cli/internal/constants"
 	"github.com/ActiveState/cli/internal/machineid"
 )
 
+type Configurable interface {
+	InstallSource() string
+}
+
 type delayedLog struct {
 	level string
 	msg   interface{}
 }
 
 var delayedLogs []delayedLog
+var rollbarCfg Configurable
```

Comment:
Nit: Why are we calling this "rollbarCfg"? This is just config, and rollbar is free to use it. Seems weird to appropriate it to rollbar by name.

Reply:
Good point.